src/vs/vscode.proposed.d.ts

```diff
@@ -2403,4 +2403,89 @@ declare module 'vscode' {
 		location?: Location;
 	}
 	//#endregion
+
+	//#region Statusbar Item Background Color (https://github.com/microsoft/vscode/issues/110214)
+
+	/**
+	 * A status bar item is a status bar contribution that can
+	 * show text and icons and run a command on click.
+	 */
+	export interface StatusBarItem {
```

Comment:
In https://github.com/microsoft/vscode/pull/111237#discussion_r529721545 @jrieken advised not replicating the existing members of an interface when proposing adding a new member.

Reply:
👍